betelgeuse.py

```diff
@@ -209,6 +221,22 @@ def test_case(path, collect_only, project):
                     test_case.update()
 
 
+@cli.command('test-results')
+@click.option(
+    '--path',
+    default='tests',
+    help='Path to the test module or directory.',
+    type=click.Path(exists=True),
```

Comment:
@elyezer wait, I'm not sure why this is showing up as something I changed. I added a new `test-results` command. The `test-run` command was already in place (and missing the arg you suggested).